src/components/views/EditUserPreferencePage.js

```diff
@@ -75,18 +75,18 @@ const EditUserPreferencePage = () =>{
         else{setMaxRange(scale(newValue))}
     };
 
-    const convertMaxRangeToDistance = (maxRange) =>{
-        if(maxRange <= 10){
-            return maxRange
+    const convertMaxRangeToDistance = (mR) => {
+        if(mR <= 10){
+            return mR
         }
-        if(maxRange <= 100){
-            return ((maxRange-10)/9) + 9
+        if(mR <= 100){
+            return ((mR-10)/9) + 9
         }
-        if(maxRange <= 1000){
-            return ((maxRange-100)/90) + 18
+        if(mR <= 1000){
+            return ((mR-100)/90) + 18
         }
         else{
-            return ((maxRange-1000)/2210) + 27
+            return ((mR-1000)/2210) + 27
```

Comment:
why shorten maxRange to mR? i would not do that, as it hurts readability but otherwise makes no difference

Reply:
agree

Reply:
@TwoDigitsOneNumber because maxRange is declared in the upper scope, and that's considered bad style^^

Reply:
@daveking7 ok, that is a good point, that way it's not confused with the react hook. I'll then merge and close it